### DISCUSSION TITLE
Options for Capitated PMPM Rates?

### DISCUSSION THREAD

**Discussion by nicholas-dahl-CTS** [CMSgov/price-transparency-guide#163] [Q&A]
Due to the fact that a capitated PM/PM rate is not specific to a submitted service (CPTs or HCPCs  or Revenue codes), but rather it is specific to member factors (age, gender, relationship), is it acceptable that the INN MRF indicate ‘capitated’ in the “Negotiation Arrangement” field for a capitated service and provide the negotiated rate (for example: a FFS rate) in the “Negotiated Rates” field that would have been paid to the provider if the service had not been capitated?

If the above is not acceptable, can the negotiated rate in the “Negotiated Rates” field for a capitated service identified as ‘Capitated’ in the “Negotiation Arrangement” field be identified as 0.00 since the provider would not be paid a rate due to the service being submitted on a claim would be identified as capitated per the provider’s agreement?

**Comment by jltang-kp** [comment]
We are planning to do something similar for our pmpm agreements with a possible companion file of pmpm rates. 

Though it's true that since the service is capitated the provider compensation for the specific service will be $0. It seems misleading to not include atleast a disclaimer that these providers are being paid on a PMPM amount.

Interestingly the example capitation file does list a non $0 negotiated rate.

https://github.com/CMSgov/price-transparency-guide/blob/master/examples/in-network-rates/in-network-rates-capitation-sample.json

**Reply by bryanbriegel** [reply]
I believe the reason the example file has financials, is the rule notes in the case of capitation: "The PMPM rate would be the negotiated rate."  too "the In-network Rate File, the plan or issuer would be required to report the negotiated rate,
which in this case is the PMPM rate, and the underlying fee schedule rate used to determine cost-sharing
liability."  though I don't see a field for this in the INN schema.
